sklearn/preprocessing/data.py

```diff
@@ -121,9 +121,8 @@ def scale(X, axis=0, with_mean=True, with_std=True, copy=True):
 
     See also
     --------
-    :class:`sklearn.preprocessing.StandardScaler` to perform centering and
-    scaling using the ``Transformer`` API (e.g. as part of a preprocessing
-    :class:`sklearn.pipeline.Pipeline`)
+    StandardScaler: Performs scaling to unit variance using the``Transformer`` API
+         (e.g. as part of a preprocessing :class:`sklearn.pipeline.Pipeline`).
```

Comment:
Can you post a screenshot of what you get after compiling these docs? We don't want the new docs to suffer from the same problem as the old docs:

![screenshot 2016-05-02 14 35 42](https://cloud.githubusercontent.com/assets/1865885/14965469/41ddcdfc-1073-11e6-8ec8-8940418118d0.png)